Synergism.css

```diff
@@ -213,118 +265,138 @@ button:focus {
 #prompt > label {
     grid-area: txt;
     text-align: center;
-    margin-left: 15px;
-    margin-right: 15px;
+    white-space: pre-line;
+    margin: 0;
 }
 
 /* Elements relating to the browser popup functions (Alert, Confirm, etc) */
 
 #ok_confirm,
 #ok_alert,
-#ok_prompt { grid-area: btn; }
+#ok_prompt {
+    grid-area: btn;
+}
 
 #cancel_confirm,
-#cancel_prompt { grid-area: btn2; }
-#prompt_text { grid-area: lbl; }
+#cancel_prompt {
+    grid-area: btn2;
+}
+
+#prompt_text {
+    font-size: 1em;
+    grid-area: lbl;
+    margin: 0 -20%;
+}
 
 #confirm > button,
 #alert > button,
 #prompt > button {
-    width: 75%;
     border: 1px solid gold;
-    margin: 0 auto;
-    height: 50%;
-    transition-duration: 0.15s;
-}
-
-#confirm > button:hover,
-#alert > button:hover,
-#prompt > button:hover {
-    background-color: #333;
 }
 
 #confirm {
     height: 100%;
     display: grid;
-    grid-template-columns: 25% 25% 25% 25%;
-    grid-template-rows: 75% 25%;
-    grid-template-areas:
-        "txt txt txt txt"
-        ". btn btn2 .";
+    grid-template-columns: auto 6em 1em 6em auto;
+    grid-template-rows: auto 2em 0.5em;
+    grid-template-areas: "txt txt txt txt txt" ". btn . btn2 ." ". . . . .";
+    outline: none;
 }
 
 #alert {
     height: 100%;
     display: grid;
-    grid-template-columns: 33.3333% 33.3333% 33.3333%;
-    grid-template-rows: 75% 25%;
-    grid-template-areas:
-        "txt txt txt"
-        ". btn .";
+    grid-template-columns: auto 8em auto;
+    grid-template-rows: auto 2em 0.5em;
+    grid-template-areas: "txt txt txt" ". btn ." ". . .";
+    outline: none;
 }
 
 #prompt {
     height: 100%;
     display: grid;
-    grid-template-columns: 25% 25% 25% 25%;
-    grid-template-rows: 5% 55% 10% 5% 25%;
-    grid-template-areas:
-        ". . . ."
-        "txt txt txt txt"
-        ". lbl lbl ."
-        ". . . ."
-        ". btn btn2 .";
+    grid-template-columns: auto 6em 2em 6em auto;
+    grid-template-rows: auto 0.5em 2em 1em 2em 0.5em;
+    grid-template-areas: "txt txt txt txt txt" ". . . . ." ". lbl lbl lbl ." ". . . . ." ". btn . btn2 ." ". . . . .";
+    outline: none;
+}
+
+.confirmBox {
+    display: flex;
+    align-items: center;
 }
 
 #resourcesrow {
     position: relative;
-    font-size: 16px;
     width: 100%;
 }
 
 #preloadContainer {
+    display: flex;
+    background: rgb(20 20 20);
     position: absolute;
     top: 0;
     left: 0;
-    z-index: 2;
-    height: 200vh;
+    z-index: 10;
+    width: 100vw;
+    height: 100vh;
     min-height: 100vh;
-    width: 100%;
-    display: flex;
     align-items: center;
+    flex-direction: column;
 }
 
 #preload {
     flex: 1;
+    width: 100vw;
+    height: 100vh;
+    min-width: 1084px;
+    min-height: 610px;
+}
+
+#preloadDeleteGame {
+    position: absolute;
```

Comment:
does this have to be absolutely positioned?

Reply:
Replaced with fixed